server/php/UploadHandler.php

```diff
@@ -424,12 +424,13 @@ protected function validate($uploaded_file, $file, $error, $index) {
             return false;
         }
         if (is_int($this->options['max_number_of_files']) &&
-                ($this->count_file_objects() >= $this->options['max_number_of_files']) &&
-                // Ignore additional chunks of existing files:
-                !is_file($this->get_upload_path($file->name))) {
+            ($this->count_file_objects() >= $this->options['max_number_of_files']) &&
+            // Ignore additional chunks of existing files:
+            !is_file($this->get_upload_path($file->name))) {
             $file->error = $this->get_error_message('max_number_of_files');
             return false;
         }
+        $img_height = 0;
```

Comment:
This should not be necessary, since `$img_height` is always set if `$img_width` is set.

See the following two lines:
https://github.com/blueimp/jQuery-File-Upload/blob/v9.30.0/server/php/UploadHandler.php#L439
https://github.com/blueimp/jQuery-File-Upload/blob/v9.30.0/server/php/UploadHandler.php#L454

For clarity however you might wanna change
```php
if (!empty($img_width)) {
```
to
```php
if (!empty($img_width) && !empty($img_height)) {
```

Reply:
Yeah that it is a much better idea. I've ran into an issue with dividing by zero on friday with $y set to 0, when it was unable to load the image due to a wrong path and then not being able to read the image size properties. I'll look to also add a check if it actually loaded the image correctly before trying to do something with it.